src/main/scala/org/globalforestwatch/layers/GrossEmissionsNodeCodes.scala

```diff
@@ -0,0 +1,92 @@
+package org.globalforestwatch.layers
+
+import org.globalforestwatch.grids.GridTile
+
+case class GrossEmissionsNodeCodes(gridTile: GridTile, model: String = "standard")
+  extends StringLayer
+    with OptionalILayer {
+  //      val model_suffix = if (model == "standard") "" else s"__$model"
+  val model_suffix: String = if (model == "standard") "standard" else s"$model"
+
+  val uri: String =
+//    s"$basePath/gfw_emissions_node_codes$model_suffix/v20200824/raster/epsg-4326/${gridTile.gridSize}/${gridTile.rowCount}/name/geotiff/${gridTile.tileId}.tif"
+    s"s3://gfw-files/flux_2_1_0/gross_emissions_node_codes/$model_suffix/${gridTile.tileId}.tif"
+
+  override val externalNoDataValue = "Not applicable"
+
+  def lookup(value: Int): String = value match {
```

Comment:
Codacy found an issue: [Method is longer than 50 lines](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)